### PR TITLE
Submodule's belongs-to reference finds wrong main module #195

### PR DESCRIPTION
**Pull request by dhuebner**
Scoping: consider sibling first when multiple candidates are available

### CODE REVIEW

**Comment by spoenemann** [comment]
The change looks good, but I get two test errors when running `./gradlew build` in Gitpod:
```
io.typefox.yang.tests.integration.GoodTests > checkDiagnostics[ietf-snmp-tls.yang] FAILED
    org.junit.ComparisonFailure at GoodTests.java:201

io.typefox.yang.tests.integration.GoodTests > checkDiagnostics[augment-super.yang] FAILED
    org.junit.ComparisonFailure at GoodTests.java:201
```
@dhuebner are these unrelated?

It would be great to have pull request validation. Could you add a GitHub workflow for that? See for example [this CI workflow](https://github.com/eclipse/openvsx/blob/master/.github/workflows/main.yml) (without the NodeJS / Yarn / Docker stuff).

**Comment by dhuebner** [comment]
> ```
> io.typefox.yang.tests.integration.GoodTests > checkDiagnostics[ietf-snmp-tls.yang] FAILED
>     org.junit.ComparisonFailure at GoodTests.java:201
> 
> io.typefox.yang.tests.integration.GoodTests > checkDiagnostics[augment-super.yang] FAILED
>     org.junit.ComparisonFailure at GoodTests.java:201
> ```
> 
> @dhuebner are these unrelated?

I successfully ran all the tests locally before pushing. I hope it doesn't depend on gradle vs. local junit testing.

 
> 
> It would be great to have pull request validation. Could you add a GitHub workflow for that? See for example [this CI workflow](https://github.com/eclipse/openvsx/blob/master/.github/workflows/main.yml) (without the NodeJS / Yarn / Docker stuff).

Will do!

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - dh-moduleRef-siblings-195
+  pull_request:
+    branches:
+      - dh-moduleRef-siblings-195
```
Comment (spoenemann) [inline .github/workflows/main.yml:9]: I think this means the branch into which the PR shall be merged, so it should be `master`.

**Comment by dhuebner** [comment]
@spoenemann 
On my Mac all tests are passing (junit and gradle). Github Action build reports 4 failing tests, you reported 2 failing tests, me (using gitpod.io) can report 1 failing test. 
In my case linker can't link to `"/interfaces/ifEntry" ` in `src/test/resources/good/augment-super.yang`

I would like to investigate this tomorrow (can't imagine it is related to my changes)

File: .github/workflows/main.yml (hunk 1 above)
Comment (dhuebner) [reply .github/workflows/main.yml:9]: > I think this means the branch into which the PR shall be merged, so it should be `master`.

I just for testing now, was not sure if it works at all. Will change both to master before we can merge

**Comment by dhuebner** [comment]
@spoenemann 
I think it can be merged now

**Review by spoenemann — APPROVED** [review]
Thanks!

**Pull request merged**